src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -912,10 +912,6 @@ internal int High
     /// </summary>
     struct VolatileBool
     {
-        public VolatileBool(bool value)
```

Comment:
Perhaps I'll make the change and submit a pull request for evaluation (which of course may or may not get included).